tests/perf/benchmark.py

```diff
@@ -0,0 +1,200 @@
+"""OTX Benchmark based on tools/experiment.py."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import os
+import glob
+import pandas as pd
+import yaml
+from pathlib import Path
+from typing import List, Optional
+
+from tests.test_suite.run_test_command import check_run
+
+
+class OTXBenchmark:
+    """Benchmark runner based on tools/experiment.py in OTX1.x.
+
+    Example:
+        >>> bm = OTXBenchmark(['random_sample1', 'random_sample2'], data_root='./data/coco')
+        >>> atss_result = bm.run('MobileNetV2-ATSS')
+        >>> yolox_result = bm.run('YOLOX-TINY')
+
+    Args:
+        datasets (List[str]): Paths to datasets relative to the data_root.
+            Intended for, but not restricted to different sampling based on same dataset.
+        data_root (str): Path to the root of dataset directories. Defaults to './data'.
+        num_epoch (int): Overrides the per-model default number of epoch settings.
+            Defaults to 0, which means no overriding.
+        num_repeat (int): Number for trials with different random seed, which would be set
+            as range(0, num_repeat). Defaults to 1.
+        train_params (dict): Additional training parameters.
+            e.x) {'learning_parameters.num_iters': 2}. Defaults to {}.
+        track_resources (bool): Whether to track CPU & GPU usage metrics. Defaults to False.
+        eval_upto (str): The last serial operation to evaluate. Choose on of ('train', 'export', 'optimize').
```

Comment:
```suggestion
        eval_upto (str): The last serial operation to evaluate. Choose one of ('train', 'export', 'optimize').
```
it may be typo?

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2742/commits/91bfec13541e70258511473e29b8b56dfed4c3d1